diff --git a/Assets/Objects/Prefabs/BarracksPanel_UI.prefab b/Assets/Objects/Prefabs/BarracksPanel_UI.prefab
index af33650..5fe1760 100644
--- a/Assets/Objects/Prefabs/BarracksPanel_UI.prefab
+++ b/Assets/Objects/Prefabs/BarracksPanel_UI.prefab
@@ -383,7 +383,7 @@ MonoBehaviour:
   m_fontColor32:
     serializedVersion: 2
     rgba: 4281479730
-  m_fontColor: {r: 1, g: 1, b: 1, a: 1}
+  m_fontColor: {r: 0.95, g: 0.9, b: 0.82, a: 1}
   m_enableVertexGradient: 0
   m_colorMode: 3
   m_fontColorGradient:
@@ -400,7 +400,7 @@ MonoBehaviour:
   m_faceColor:
     serializedVersion: 2
     rgba: 4294967295
-  m_fontSize: 40
+  m_fontSize: 26
   m_fontSizeBase: 40
   m_fontWeight: 400
   m_enableAutoSizing: 0
@@ -1350,7 +1350,7 @@ MonoBehaviour:
   m_fontColor32:
     serializedVersion: 2
     rgba: 4294967295
-  m_fontColor: {r: 1, g: 1, b: 1, a: 1}
+  m_fontColor: {r: 0.95, g: 0.9, b: 0.82, a: 1}
   m_enableVertexGradient: 0
   m_colorMode: 3
   m_fontColorGradient:
@@ -1367,7 +1367,7 @@ MonoBehaviour:
   m_faceColor:
     serializedVersion: 2
     rgba: 4294967295
-  m_fontSize: 50
+  m_fontSize: 26
   m_fontSizeBase: 50
   m_fontWeight: 400
   m_enableAutoSizing: 0
diff --git a/Assets/Objects/Prefabs/SettingsPanel_UI.prefab b/Assets/Objects/Prefabs/SettingsPanel_UI.prefab
index 785d247..631f29d 100644
--- a/Assets/Objects/Prefabs/SettingsPanel_UI.prefab
+++ b/Assets/Objects/Prefabs/SettingsPanel_UI.prefab
@@ -113,7 +113,7 @@ MonoBehaviour:
   m_fontColor32:
     serializedVersion: 2
     rgba: 4294967295
-  m_fontColor: {r: 1, g: 1, b: 1, a: 1}
+  m_fontColor: {r: 0.95, g: 0.9, b: 0.82, a: 1}
   m_enableVertexGradient: 0
   m_colorMode: 3
   m_fontColorGradient:
@@ -130,7 +130,7 @@ MonoBehaviour:
   m_faceColor:
     serializedVersion: 2
     rgba: 4294967295
-  m_fontSize: 35
+  m_fontSize: 26
   m_fontSizeBase: 35
   m_fontWeight: 400
   m_enableAutoSizing: 0
diff --git a/Assets/Objects/Prefabs/ShopPanel_UI.prefab b/Assets/Objects/Prefabs/ShopPanel_UI.prefab
index 2b0ef35..014f915 100644
--- a/Assets/Objects/Prefabs/ShopPanel_UI.prefab
+++ b/Assets/Objects/Prefabs/ShopPanel_UI.prefab
@@ -960,7 +960,7 @@ MonoBehaviour:
   m_fontColor32:
     serializedVersion: 2
     rgba: 4281479730
-  m_fontColor: {r: 1, g: 1, b: 1, a: 1}
+  m_fontColor: {r: 0.95, g: 0.9, b: 0.82, a: 1}
   m_enableVertexGradient: 0
   m_colorMode: 3
   m_fontColorGradient:
@@ -977,7 +977,7 @@ MonoBehaviour:
   m_faceColor:
     serializedVersion: 2
     rgba: 4294967295
-  m_fontSize: 30
+  m_fontSize: 26
   m_fontSizeBase: 30
   m_fontWeight: 400
   m_enableAutoSizing: 0
